### PR TITLE
Add MobileViT-XS to MCT-model garden

### PR DESCRIPTION
**Pull request by lapid92**
## Pull Request Description:
Add MobileViT-XS to MCT-model garden
Add MobileViT-XS tutorial

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/notebooks/pytorch/ptq/pytorch_mobilevit_xs_for_imx500.ipynb [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+   "source": [
+    "!mkdir imagenet\n",
+    "!mkdir imagenet/val\n",
+    "!wget https://image-net.org/data/ILSVRC/2012/ILSVRC2012_img_val.tar\n",
+    "!mv ILSVRC2012_img_val.tar imagenet/val\n",
+    "!tar -xvf imagenet/val/ILSVRC2012_img_val.tar -C imagenet/val\n",
+    "!rm -f imagenet/val/ILSVRC2012_img_val.tar\n",
+    "# We follow https://github.com/pytorch/examples/blob/main/imagenet/extract_ILSVRC.sh here\n",
+    "# get script from soumith and run; this script creates all class directories and moves images into corresponding directories\n",
+    "!cd imagenet/val/ && wget -qO- https://raw.githubusercontent.com/soumith/imagenetloader.torch/master/valprep.sh | bash\n",
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/pytorch/ptq/pytorch_mobilevit_xs_for_imx500.ipynb:109]: consider to take the code from here, looks more elegant.. https://github.com/sony/model_optimization/blob/main/tutorials/notebooks/keras/ptq/example_keras_imagenet.ipynb

File: tutorials/notebooks/pytorch/ptq/pytorch_mobilevit_xs_for_imx500.ipynb [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "084c2b8b-3175-4d46-a18a-7c4d8b6fcb38",
+   "metadata": {},
+   "source": [
+    "## Model Quantization\n",
+    "\n",
+    "### Download a Pre-Trained Model \n",
+    "\n",
+    "We begin by loading a pre-trained [MobileViT-XS](https://huggingface.co/SSI-DNN/pytorch_mobilevit_xs) model. This implementation is based on [Timm](https://github.com/huggingface/pytorch-image-models) and includes a slightly modified version of timm/Attention module that was adapted for model quantization. For further insights into the model's implementation details, please refer to [MCT Models Garden - yolov8](https://github.com/sony/model_optimization/tree/main/tutorials/mct_model_garden/models_pytorch/mobilevit_xs).  "
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/pytorch/ptq/pytorch_mobilevit_xs_for_imx500.ipynb:122]: Maybe add a few words what are the adjustments (batch issue..)

File: tutorials/notebooks/pytorch/ptq/pytorch_mobilevit_xs_for_imx500.ipynb [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+    "representative_dataset_gen = get_representative_dataset(n_iter=n_iters,\n",
+    "                                                        dataset_loader=representative_dataset)\n",
+    "\n",
+    "# Set IMX500-v1 TPC\n",
+    "tpc = mct.get_target_platform_capabilities(fw_name=\"pytorch\",\n",
+    "                                           target_platform_name='imx500',\n",
+    "                                           target_platform_version='v1')\n",
+    "\n",
+    "# Specify the necessary configuration for mixed precision quantization. To keep the tutorial brief, we'll use a small set of images and omit the hessian metric for mixed precision calculations. It's important to be aware that this choice may impact the resulting accuracy. \n",
+    "mp_config = mct.core.MixedPrecisionQuantizationConfig(num_of_images=5,\n",
```
Comment (Idan-BenAmi) [inline tutorials/notebooks/pytorch/ptq/pytorch_mobilevit_xs_for_imx500.ipynb:214]: check if compression is required..

**Review by Idan-BenAmi — APPROVED** [review]
check the comments
